code/datums/traits/negative.dm

```diff
@@ -370,3 +370,20 @@
 		if(prob(1))
 			new/obj/item/reagent_containers/food/snacks/spaghetti/pastatomato(get_turf(H)) //now that's what I call spaghetti code
 
+/datum/quirk/replicant_eat_that
+	name = "Replicant Snob"
+	desc = "You find yourself greatly disliking anything made by a replicator. It's the Old Fashion way, or no way!"
```

Comment:
"It's the Old Fashion way, or no way!" sounds a little off to me. Perhaps it should be "It's the Old Fashioned way, or no way!"